### PR TITLE
Nomi/prod fixes

### LINKED ISSUE #216: Correct node name section
###  Documentation feedback

 
- Page title: CrateDB multi-node setup
- Page URL: https://crate.io/docs/crate/howtos/en/latest/scaling/multi-node-setup.html
- Source: https://github.com/crate/crate-howtos/blob/master/docs/scaling/multi-node-setup.rst

---

"To do this, you must be able to refer to nodes by name."

this isn't true. you can use node name, fully-qualified hostname, or IP address

this document should be updated to account for the fact that some people may want to configure a cluster using hostnames or IP addresses without setting node names. this allows setting up a cluster with the same `crate.yaml` file shared between all nodes

### LINKED ISSUE #217: Update node name config section for shared config setups
###  Documentation feedback

 
- Page title: Going into production
- Page URL: https://crate.io/docs/crate/howtos/en/latest/going-into-production.html
- Source: https://github.com/crate/crate-howtos/blob/master/docs/going-into-production.rst

---

per https://github.com/crate/crate-howtos/issues/216, "this document should be updated to account for the fact that some people may want to configure a cluster using hostnames or IP addresses without setting node names. this allows setting up a cluster with the same crate.yaml file shared between all nodes"

### LINKED ISSUE #218: Fix "node.name" reference
###  Documentation feedback

 
- Page title: CrateDB multi-node setup
- Page URL: https://crate.io/docs/crate/howtos/en/latest/scaling/multi-node-setup.html
- Source: https://github.com/crate/crate-howtos/blob/master/docs/scaling/multi-node-setup.rst

---

under the cluster name section, we reference the "node.name" setting. this should reference the "cluster.name" setting

### LINKED ISSUE #220: Rename section "Clustering"
###  Documentation feedback

 
- Page title: Scaling
- Page URL: https://crate.io/docs/crate/howtos/en/latest/scaling/index.html
- Source: https://github.com/crate/crate-howtos/blob/master/docs/scaling/index.rst

---

I think it would make more sense to rename this section "clustering". scaling up and scaling down a cluster are naturally a part of clustering

I think that new users are going to want to learn how to "cluster" before they want to learn how to "scale". so I think the docs should reflect that priority in terms of what terminology is used

### PR DESCRIPTION
**Pull request by mechanomi**
- Fix node.name reference (fixes #218)

- Improve docs for node names, DNS labels, and shared config setups (fixes #216, fixes #217)

- Rename Scaling section to Clustering (fixes #220)

### CODE REVIEW

**Comment by mechanomi** [comment]
@proddata I took your feedback about using hostnames for cluster bootstrapping and I have reworked the multi-node setup docs and going into production docs. can you take a look and see what you think?

File: docs/going-into-production.rst [hunk 1]
```diff
@@ -102,143 +154,149 @@ their type. For instance:
...
 
 .. code-block:: yaml
 
     node.master: true
     node.data: false
 
-A good name for this node might be:
+A good DNS label for this node might be ``node-03-m``.
+
+Here, ``node`` is used as base label with a sequence number of ``02``. Every
```
Comment (infoverload) [inline docs/going-into-production.rst:243]: ``02`` should be ``03``

Comment (mechanomi) [reply docs/going-into-production.rst]: pushed a fixup. thx @infoverload

Comment (proddata) [reply docs/going-into-production.rst]: The node name isn't necessary a DNS label is it?

e.g.:
node name: node-03-m
hostname: node03.cluster.crate.io

**Comment by mechanomi** [comment]
@proddata pushed a fixup. can you re-review please?

**Pull request merged**